docs/modules/release-notes/pages/all-releases/4_15.adoc

```diff
@@ -0,0 +1,91 @@
+= Kobiton 4.15 release notes
+:navtitle: Kobiton 4.15 release notes
+
+_September 14, 2024_
+
+== New OS support
+
+This release adds support for the following:
+
+* iOS/iPadOS 18 beta 8
+
+[NOTE]
+For iOS/IPadOS 18 Beta 8, the cleanup policy does not clear signed-in accounts in the Mail app and Safari browser cache.
+
+== Restart a device for users without ADMIN role
+
+Before this release, only users with the `ADMIN` predefined role could restart a private/local device, and they could only do this from the Device Management page.
+
+With this release, users with the appropriate permission can restart a device by selecting *Restart* via the *Device List* page.
+
+[NOTE]
+====
+
+* Users without ADMIN role must be assigned a custom role with the permission `org_setting.reboot_private_device` to use this feature.
+
+* Non-admin users with restart permission may only restart devices they can access via their assigned teams.
+
+* Only devices with *Available* status or *Retained* by the current user can be restarted from the device details.
+
+====
+
+== Option to enable/disable unlimited retention for devices
+
+This release introduces the option to enable/disable unlimited retention of devices for an org. In *Settings → Other Settings*, we added the checkbox: *Allow unlimited retention*. Check or uncheck this box, then choose *Save* to enable/disable unlimited retention of devices.
+
+[NOTE]
+
+====
+
+* To access the *Other Settings* page, the user must be assigned a role with the permission `org_setting.modify`.
+
+* The *Allow unlimited retention* checkbox applies to the whole org. By default, this checkbox is checked.
+
+====
+
+==  Monitor number of device offline errors in the last hour
```

Comment:
Is this working for Cloud now? If not, please comment out.

Reply:
Almost forgot. I will remove it.